src/main/java/org/isf/lab/gui/LabNew.java

```diff
@@ -751,20 +751,20 @@ private JTable getJTableExams() {
 			listSelectionModel.addListSelectionListener(new ListSelectionListener() {
 				
 				public void valueChanged(ListSelectionEvent e) {
-					// Check that mouse has been released.
```

Comment:
These look like just formatting changes that actually makes things out of line; see image:
![image](https://user-images.githubusercontent.com/1105445/118013564-3e458180-b320-11eb-8395-89f63d897b4c.png)

If you really want to change this then I'd suggest using a lambda:
![image](https://user-images.githubusercontent.com/1105445/118013936-a98f5380-b320-11eb-9412-7618050f0f83.png)